DOCS-2541: Move component APIs (#3504)
viamrobotics/docs: docs/appendix/apis/components/base.md

---
title: "Base API"
linkTitle: "Base"
weight: 20
type: "docs"

> description: "The base API allows you to give commands to your base components for moving all configured components attached to a platform as a whole without needing to send commands to individual components."

Reviewer: ```suggestion
description: "Commands for moving all configured components attached to a mobile platform as a whole without needing to send commands to individual components."
```
